apps/fabric-website/src/pages/Controls/ControlsAreaPage.tsx

```diff
@@ -22,4 +22,6 @@ const ControlsAreaPageBase: React.StatelessComponent<IControlsPageProps> = props
   return <Page subTitle={getSubTitle(props.platform)} jsonDocs={jsonDocs} {...props} />;
 };
 
-export const ControlsAreaPage: React.StatelessComponent<IControlsPageProps> = withPlatform<Platforms>(ControlsAreaPageBase);
+export const ControlsAreaPage: React.StatelessComponent<IPageProps<Platforms>> = (props: IPageProps<Platforms>) => (
```

Comment:
I'm confused why `withPlatform` wasn't working as expected, since in theory this new code should be the exact same thing the HOC was doing. If you can figure out how to fix `withPlatform`, that seems like a cleaner approach.

Reply:
I am too. I'm hypothesizing it's something to do with being pulled out of node_modules instead of locally possibly causing a de-sync in context objects.

Reply:
Is it possible that this was something you tried changing earlier in the process of debugging that didn't turn out to be the actual root issue? Or maybe it could be an issue with either how the provider is being created in Site, or how the current platform is being updated (like it's not being updated in a way that causes the provider to update its value prop)?